### PR TITLE
Remove Span.Status per OTEP-134

### LINKED ISSUE #706: Remove Span.Status from API
It was proposed in https://github.com/open-telemetry/oteps/pull/123#issuecomment-656788771 and subsequent discussion that we should drop `Span.Status` from the spec. If there is an agreement on that, I can create a PR.

### PR DESCRIPTION
**Pull request by iNikem**
Fixes #706

## Changes

Remove `Span.Status` as per [OTEP-134](https://github.com/open-telemetry/oteps/pull/134)

### CODE REVIEW

File: specification/trace/semantic_conventions/rpc.md [hunk 1]
```diff
@@ -89,17 +89,13 @@ Note that _method_ in this context is about the called remote procedure and _not
 
 For remote procedure calls via [gRPC][], additional conventions are described in this section.
 
-`rpc.system` MUST be set to `"grpc"`.
+| Attribute name |                          Notes and examples                            | Required? |
+| -------------- | ---------------------------------------------------------------------- | --------- |
+| `rpc.system`   | MUST be set to `"grpc"`       | Yes |
+| `rpc.status`  | Canonical status code of the gRPC call's response    | Yes |
```
Comment (Oberon00) [inline specification/trace/semantic_conventions/rpc.md:95]:
```suggestion
| `rpc.grpc.status`  | Canonical status code of the gRPC call's response    | Yes |
```

File: spec-compliance-matrix.md [hunk 2]
```diff
@@ -35,7 +35,6 @@ status of the feature is not known.
 |End                                           | + | +  | +     | +    | +  | +    | + | +  |   | +  |
 |End with timestamp                            | + | +  | +     | +    | +  | +    | + | -  |   | +  |
 |IsRecording                                   | + | +  | +     | +    | +  | +    | + |    |   | +  |
-|Set status                                    | + | +  | +     | +    | +  | +    | + | +  |   | +  |
```
Comment (arminru) [inline spec-compliance-matrix.md:38]: We could keep this and add something like `(REMOVED)` to keep track of its removal process (until all SIGs are done).

File: specification/trace/semantic_conventions/rpc.md [hunk 3]
```diff
@@ -13,14 +13,13 @@ This document defines how to describe remote procedure calls
...
 - [gRPC](#grpc)
-  * [Status](#status)
   * [Events](#events)
 
 <!-- tocstop -->
 
 ## Common remote procedure call conventions
 
-A remote procedure calls is described by two separate spans, one on the client-side and one on the server-side.
+A remote procedure calls are described by two separate spans, one on the client-side and one on the server-side.
```
Comment (arminru) [inline specification/trace/semantic_conventions/rpc.md:22]:
```suggestion
A remote procedure call is described by two separate spans, one on the client-side and one on the server-side.
```

**Review by MikeGoldsmith — APPROVED** [review]

**Review by anuraaga — APPROVED** [review]

**Review by Oberon00 — APPROVED** [review]
Approving in the sense of "I (and Dynatrace) would be OK with this solution" (even without replacement, as http.status_code et al is enough for us).

Replacing status with unspecified/ok/error would also be fine with me (we would probably still not use it at Dynatrace though as we prefer more specific error information).

**Comment by tedsuo** [comment]
@iNikem do you mind closing this PR, now that we have OTEP 136? Related issue: https://github.com/open-telemetry/opentelemetry-specification/issues/965.

**Comment by iNikem** [comment]
Closed in favor of open-telemetry/oteps#134

**Pull request closed without merging**